apps/web/app/modules/Admin/Courses/LessonAssigner/LessonCard.tsx

```diff
@@ -51,7 +51,7 @@ export function LessonCard({ lesson }: LessonCardProps) {
       />
       <div>
         <h4 className="font-medium">{lesson.title}</h4>
-        <Viewer className="line-clamp-2" content={lesson.description} />
+        <Viewer className="line-clamp-2" content={lesson.description || ""} />
```

Comment:
If I understand correctly, you have already defined this value at the api level as a string

Reply:
It's also about old data, it was quickest fix, we will get rid of this in the future when this view will be removed